dbt/profiles.yml

```
@@ -26,3 +26,51 @@ dbt-civis:
       project: "{{ env_var('GCP_PROJECT_ID') }}"
       schema: "{{ env_var('DBT_SCHEMA') }}"
       token: "{{ env_var('DBT_ENV_SECRET_GCP_ACCESS_TOKEN') }}"
+
+
+
+#############################################
+########## dbt in Civis Studios #############
+#############################################
+
+# Civis Studios have slightly different environment variables than
+# dbt scripts in Platform (see the dbt-civis profile above for comparison).
+#
+# The following profiles demonstrate the necessary changes to use the Civis Studio environment:
+# For a database parameter named "civis_consumers":
+#  "params": [
+#    {
+#        "name": "civis_consumers",
+#        "description": null,
+#        "type": "database",
+#        "value": "{\"credential\":1234,\"database\":5678}"
+#    }, {
+#        "name": "dbt_schema",
+#        "description": null,
+#        "type": "string",
+#        "value": "my_dbt_schema"
+#    }
+#  ]
+# , the corresponding environment variables in Civis Studios are:
+civis-studio:
+  target: redshift_adapter
+  outputs:
+    redshift_adapter:
+      type: "{{ env_var('CIVIS_CONSUMER_TYPE') }}"
+      host: "{{ env_var('CIVIS_CONSUMER_HOST') }}"
+      port: "{{ env_var('CIVIS_CONSUMER_PORT') | as_number }}"
+      user: "{{ env_var('CIVIS_CONSUMER_CREDENTIAL_USERNAME') }}" # note the _CREDENTIAL
+      pass: "{{ env_var('DBT_ENV_SECRET_CIVIS_CONSUMER_CREDENTIAL_PASSWORD') }}" # note the _CREDENTIAL
+      dbname: "{{ env_var('CIVIS_CONSUMER_DATABASE') }}" # _DATABASE instead of _NAME
+      schema: "{{ env_var('DBT_SCHEMA') }}"
+      threads: 4
+
+    bigquery_oauth_adapter:
+      threads: 4 # Must be a value of 1 or greater
+      type: "{{ env_var('CIVIS_CONSUMER_TYPE') }}"
+      method: "{{ env_var('CIVIS_CONSUMER_GCP_AUTH_METHOD') }}"
+      project: "{{ env_var('CIVIS_CONSUMER_GCP_PROJECT_ID') }}"
+      schema: "{{ env_var('DBT_SCHEMA') }}"
+      token: "{{ env_var('CIVIS_CONSUMER_DBT_ENV_SECRET_GCP_ACCESS_TOKEN') }}"
```

The env secret part comes before the db name
```suggestion
      token: "{{ env_var('DBT_ENV_SECRET_CIVIS_CONSUMER_GCP_ACCESS_TOKEN') }}"
```